src/main/resources/view/HelpWindow.fxml

```diff
@@ -10,7 +10,7 @@
 
 <!-- TODO: set a more appropriate initial size -->
```

Comment:
By the way, is this comment still relevant?

Reply:
Kinda, we're still not setting a fixed size (mainly because I foresee students just replacing the link and having the label overflow).

Reply:
> mainly because I foresee students just replacing the link and having the label overflow

Assuming that students do something like:
```diff
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
+    public static final String USERGUIDE_URL = "https://my.new.website/addressbook-level3/UserGuide.html";
```

As long as the label contents are set before the window is first shown, the window will adapt its size to the size of the label, no?

Reply:
> As long as the label contents are set before the window is first shown, the window will adapt its size to the size of the label, no?

Otherwise, the call to `root.sizeToScene()` *shouldn't* be removed?

Reply:
I was referring to a fixed size like in MainWindow where the minHeight and minWidth are defined. I do agree that we can completely do without and use the computed sizes though. I'll just remove the comment for now.